pipeline.py

```diff
@@ -0,0 +1,107 @@
+                             "objects")
+    parser.add_argument("json_output_path", metavar="json-output-path",
+                        help="Path to a JSON file to write TracedData for final analysis file to")
+    parser.add_argument("icr_output_dir", metavar="icr-output-dir",
+                        help="Directory to write CSV files to, each containing 200 messages and message ids for use " 
+                             "in inter-code reliability evaluation"),
+    parser.add_argument("prev_coded_dir_path", metavar="prev-coded-dir-path",
+                        help="Directory containing Coda files generated by a previous run of this pipeline. "
+                             "New data will be appended to these files.")
+
+    parser.add_argument("coded_dir_path", metavar="coded-dir-path",
+                        help="Directory to write coded Coda files to")
+
+    args = parser.parse_args()
+
+    drive_credentials_path = None
+    csv_by_message_drive_path = None
+    csv_by_individual_drive_path = None
+    production_csv_drive_path = None
+
+    drive_upload = args.drive_upload is not None
+    if drive_upload:
+        drive_credentials_path = args.drive_upload[0]
+        csv_by_message_drive_path = args.drive_upload[1]
+        csv_by_individual_drive_path = args.drive_upload[2]
+        production_csv_drive_path = args.drive_upload[3]
+
+    user = args.user
+    pipeline_configuration_file_path = args.pipeline_configuration_file_path
+
+    phone_number_uuid_table_path = args.phone_number_uuid_table_path
+    s02e01_input_path = args.s02e01_input_path
+    
+    json_output_path = args.json_output_path
+    icr_output_dir = args.icr_output_dir
+    prev_coded_dir_path = args.prev_coded_dir_path
+    coded_dir_path = args.coded_dir_path
+
+    message_paths = [s02e01_input_path]
+
+    # Load the pipeline configuration file
+    print("Loading Pipeline Configuration File...")
+    with open(pipeline_configuration_file_path) as f:
+        pipeline_configuration = PipelineConfiguration.from_configuration_file(f)
+
+    # Load phone number <-> UUID table
+    print("Loading Phone Number <-> UUID Table...")
+    with open(phone_number_uuid_table_path, "r") as f:
+        phone_number_uuid_table = PhoneNumberUuidTable.load(f)
+
+    # Load messages
+    messages_datasets = []
+    for i, path in enumerate(message_paths):
+        print("Loading Episode {}/{}...".format(i + 1, len(message_paths)))
+        with open(path, "r") as f:
+            messages_datasets.append(TracedDataJsonIO.import_json_to_traced_data_iterable(f))
+    
+    # Add survey data to the messages
+    print("Combining Datasets...")
+    data = CombineRawDatasets.combine_raw_datasets(user, messages_datasets, [])
+
+    print("Translating Rapid Pro Keys...")
+    data = TranslateRapidProKeys.translate_rapid_pro_keys(user, data, pipeline_configuration, prev_coded_dir_path)
+    
+    print("Auto Coding Messages...")
```

Comment:
Comment is auto-coding messages but the call is to auto-coding surveys

Reply:
Fixed